.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: c
+dist: trusty
```

Comment:
No relationship between `dist` and `DISTRO`. Travis has 2 different "base" systems, I'm just selecting one of them.